packages/styles/src/breadcrumb/_breadcrumb.scss

```diff
@@ -0,0 +1,34 @@
+@import "../core/_dev";
+
+.as-breadcrumb {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+
+  &__item {
+    color: var(--as-color-type-01, #{$color-type-01});
+    font: $font-body;
+    cursor: default;
+
+    &::after {
+      content: '›';
+      margin: 0 6px;
+      color: var(--as-color-primary, #{$color-primary});
+      font-size: 12px;
+    }
+
+    &:last-child::after {
+      display: none;
+    }
+  }
+
+  &__link {
```

Comment:
I think we could just overwrite the `a` element styles so the user has to write less code